packages/sync-server/src/app-sync.test.ts

```diff
@@ -536,6 +540,8 @@ describe('/list-user-files', () => {
 
   it('returns a list of user files for an authenticated user', async () => {
     createUser('fileListAdminId', 'admin', ADMIN_ROLE, 1);
+    onTestFinished(() => deleteUser('fileListAdminId'));
```

Comment:
When re-running tests in VS Code, it doesn't re-tear-down the globalSetup file, which means that the same database is reused. This makes sure that the database is cleaned up for future runs.